tests/test_context_managers.py

```diff
@@ -125,6 +125,19 @@ def test_cd_prefix():
         eq_(command_out, 'cd %s >/dev/null && foo' % some_path)
 
 
+#
+# prefix $SHELL
+#
+
+def test_prefix_shell_variable():
+    """
+    prefix should set the $SHELL varible for consistent behavior
```

Comment:
this whole issue really has nothing at all to do with `prefix()`

Reply:
Oops, you're right. Forgot to move that. :(